packages/php/email-editor/src/Integrations/Core/Renderer/Blocks/class-embed.php

```
@@ -0,0 +1,297 @@
+		return '';
+	}
+
+	/**
+	 * Extract provider URL from block attributes or content.
+	 *
+	 * @param array  $attr Block attributes.
+	 * @param string $block_content Block content.
+	 * @param string $provider Provider name.
+	 * @return string Provider URL or empty string.
+	 */
+	private function extract_provider_url( array $attr, string $block_content, string $provider ): string {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$provider'. (undefined)

(UnusedFormalParameter)